spec/message.abnf

```diff
@@ -21,7 +21,7 @@ annotation = (function *(s option)) / reserved / private-use
 literal = quoted / unquoted
 variable = "$" name
 function = (":" / "+" / "-") name
-option = name [s] "=" [s] (literal / variable)
+option = name [[s] "=" [s] (literal / variable)]
 
 ; reserved keywords are always lowercase
```

Comment:
We should reserve the keyword `true` if this goes in?

Reply:
I don't think that would be required? The resolved value of a un-valued option doesn't actually show up in the syntax.